sirepo/template/ml.py

```diff
@@ -736,6 +744,43 @@ def _get_fit_report(report, x_vals, y_vals):
     return param_vals, param_sigmas, plots
 
 
+def _get_remote_data(url, headers_only):
+    import os
+    import urllib
+    from urllib import parse
+    from urllib import request
+
+    filename = os.path.basename(urllib.parse.urlparse(url).path)
+    try:
+        with urllib.request.urlopen(url) as r:
+            if headers_only:
+                return PKDict(headers=_header_str_to_dict(r.headers))
+            chunk_size = 1024 * 1024
+            with open(
+                _SIM_DATA.lib_file_write_path(
+                    _SIM_DATA.lib_file_name_with_model_field(
+                        "dataFile", "file", filename
+                    )
+                ),
+                "wb",
+            ) as f:
+                while True:
+                    c = r.read(chunk_size)
+                    if not c:
+                        break
+                    f.write(c)
+    except urllib.error.HTTPError as e:
```

Comment:
Why are we return this error and not other errors?

Reply:
[57d671f](https://github.com/radiasoft/sirepo/pull/4671/commits/57d671f1c5317c57d30d5462e48538d24a099224)